DependencyInjection/SncRedisExtension.php

```diff
@@ -150,9 +150,7 @@ protected function loadClient(array $client, ContainerBuilder $container)
      */
     protected function loadPredisClient(array $client, ContainerBuilder $container)
     {
-        if (null === $client['options']['cluster']) {
-            unset($client['options']['cluster']);
-        } else {
+        if (true === $client['options']['cluster']) {
```

Comment:
it should test if the value is different than `false` instead because Predis is using string to configure cluster

see https://github.com/nrk/predis#client-configuration